setup.cfg

```
@@ -0,0 +1,73 @@
+[metadata]
+name = stepwise
+description = A Python-based Trajectory Optimization Framework
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = Stefano Dafarra
+author_email = stefano.dafarra@iit.it
+license = BSD-2
+license_files = LICENSE
+platforms = any
+url = https://github.com/ami-iit/stepwise
+
+project_urls =
+    Changelog = https://github.com/ami-iit/stepwise/releases
+    Source = https://github.com/ami-iit/stepwise
+    Tracker = https://github.com/ami-iit/stepwise/issues
+
+keywords =
+    trajectory
+    optimization
+    robots
+    humanoids
+    quadrupeds
+    kinematics
+    dynamics
+    contacts
+
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Framework :: Robot Framework
+    Intended Audience :: Science/Research
+    Intended Audience :: Developers
+    Intended Audience :: Education
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Operating System :: POSIX :: Linux
+    Operating System :: MacOS
+    Operating System :: Microsoft :: Windows
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: Implementation :: CPython
+    Topic :: Games/Entertainment :: Simulation
+
+[options]
+zip_safe = False
+packages = find:
+package_dir =
+    =src
+python_requires = >=3.8
+install_requires =
+    coloredlogs
+    mashumaro
+    numpy
+    packaging
+    scipy
+    xmltodict
+
+[options.extras_require]
+style =
+    black
+    isort
+pptree =
+    pptree
+all =
+    %(style)s
+    %(pptree)s
```

Since there's no code yet, I'd initialize only styling dependencies as follows:

```suggestion
install_requires =

[options.extras_require]
style =
    black
    isort
all =
    %(style)s
```

---

Done in https://github.com/ami-iit/hippopt/pull/1/commits/7cff695da46883033771f59b12996a93269852d7